arch/X86/X86IntelInstPrinter.h

```diff
@@ -0,0 +1,19 @@
+/* Capstone Disassembly Engine */
+/* By Marius Melzer <marius@rasumi.net>, 2019 */
+
+#ifndef CS_X86_INTELINSTPRINTER_H
+#define CS_X86_INTELINSTPRINTER_MODULE_H
+
+#include "../../utils.h"
+
+void printSrcIdx8(MCInst *MI, unsigned OpNo, SStream *O);
```

Comment:
these functions are all local, so we can just turn them static, like in this commit https://github.com/aquynh/capstone/commit/9cd94caa9081bbf79f99c84d748bde0a165efe05